app/controllers/miq_ae_class_controller.rb

```diff
@@ -2819,10 +2819,18 @@ def git_refresh
     git_repo.reload
     @branch_names = git_repo.git_branches.collect(&:name)
     @tag_names = git_repo.git_tags.collect(&:name)
+    @ref_type = MiqAeDomain.find(params[:id]).ref_type
+    if @ref_type == "tag"
+      @ref_type = "Tag"
+    elsif @ref_type == "branch"
+      @ref_type = "Branch"
+    end
```

Comment:
I don't understand the need to captilize and carry that capitalization.  I would think we can leave it lowercase cause that's the backend value.